x-pack/plugins/observability/public/hooks/use_has_data_context.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { useContext } from 'react';
+import { HasDataContext } from '../context/has_data_context';
+
+export function useHasDataContext() {
```

Comment:
I don't think the `Context` suffix is necessary here. IMO, `useHasData` is descriptive enough.